src/controller/AutoCommissioner.cpp

```diff
@@ -298,6 +298,19 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStage(CommissioningStag
 
 CommissioningStage AutoCommissioner::GetNextCommissioningStageNetworkSetup(CommissioningStage currentStage, CHIP_ERROR & lastErr)
 {
+    if (IsSecondaryNetworkSupported())
+    {
+        if (IsTriedSecondaryNetwork())
+        {
+            // Try secondary network interface.
```

Comment:
I don't get it.  The test above passed, so we _have_ tried it already....  Unless the naming is just completely off somehow?